src/items/MagicBox.java

```diff
@@ -0,0 +1,30 @@
+package items;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+public class MagicBox extends Item {
+	private static ArrayList<Item> magicBoxDrop = new ArrayList<Item>();
+	public static void initDropBox() {
+		magicBoxDrop.add(Sword.sword1);
```

Comment:
don't looks good. I think that will be better when sword1, sword2 etc you will change to array.